README.md

```diff
@@ -1,4 +1,76 @@
-# ci-management
+# Continuous Integration for JQuery at LF
 
-This repo contains configuration files for Jenkins jobs for the JQuery
-project
+This repository has templates that generate jobs in the Linux Foundation Jenkins instance
+using the Jenkins Job Builder. For more information about that tool please see:
+
+    https://docs.openstack.org/infra/system-config/jjb.html
+
+## Template design
+
+Jenkins jobs are generated from standard Global JJB templates and from custom-to-project
+JJB templates.
+
+### Global JJB templates
+
+Global JJB jobs originated with the Open Daylight project and reflect CI/CD design choices
+made there. Global JJB templates are used as much as possible in JQuery. As of this writing
+this includes CLM, RST/RTD documentation, javadoc, simple Maven/Java jar jobs and some Python
+jobs.
+
+Documentation of Global JJB templates can be found here:
+
+    http://global-jjb.releng.linuxfoundation.org/en/latest/index.html
+
+## Testing the templates
+
+These instructions explain how to test the JQuery templates using the Jenkins sandbox.
+This catches errors before submitting the changes as Gerrit reviews. Prerequisites:
+
+Install the Jenkins job builder:
+
+    pip install --user jenkins-job-builder
+
+Check out the global JJB templates submodule within this repo:
+
+    git submodule update --init
+
+### Test Locally
+
+Check sanity by running the Jenkins job-builder script in this directory:
+
+    jenkins-jobs test -r jjb
+
+### Deploy the templates to the Jenkins sandbox
+
+Login (after requesting membership in group jquery-jenkins-sandbox-access) at the Jenkins sandbox:
```

Comment:
A link to https://docs.releng.linuxfoundation.org/en/latest/helpdesk.html should be put in here to help direct people to the support system for requesting sandbox access